internal/rbd/controllerserver.go

```diff
@@ -20,16 +20,16 @@ import (
 	"context"
 	"errors"
 
-	csicommon "github.com/ceph/ceph-csi/internal/csi-common"
-	"github.com/ceph/ceph-csi/internal/journal"
-	"github.com/ceph/ceph-csi/internal/util"
-
-	librbd "github.com/ceph/go-ceph/rbd"
 	"github.com/container-storage-interface/spec/lib/go/csi"
 	"github.com/kubernetes-csi/csi-lib-utils/protosanitizer"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	klog "k8s.io/klog/v2"
+
+	csicommon "github.com/ceph/ceph-csi/internal/csi-common"
```

Comment:
this is not the standard we follow see https://github.com/ceph/ceph-csi/blob/master/docs/coding.md#imports

Reply:
Some auto plugin stuff rearranging it.. will revert that..